tests/e2e-leg-9/sandbox-restart/35-assert.yaml

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright [2021-2024] Open Text.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
```

Comment:
Is this configMap necessary to verify? and do we need this step in other places where we create sandboxes?

Reply:
This configmap is generated for each sandbox that is created. We don't need it in every place we create sandboxes.